dlt/cli/magics.py

```diff
@@ -0,0 +1,286 @@
+
+
+from dlt.common.runtime.exec_info import is_notebook, is_ipython
+from dlt.cli import echo as fmt, echo
+
+import typing as t  # type: ignore
```

Comment:
I know this is PITA but if you could place the standard imports over the dlt imports it would be awesome (we'll add isort soon)